src/mapml-viewer.js

```diff
@@ -194,6 +194,29 @@ export class MapViewer extends HTMLElement {
         }
       }, 0);
     }
+
+    let scaleinitial = this.shadowRoot.querySelector(".leaflet-container");
```

Comment:
make sure all the new variables you create are lower camelCase to stay consistent with the code base.